packages/playground/scripts/hmrPlugin/fineTune.ts

```diff
@@ -0,0 +1,57 @@
+import { init, parse } from 'es-module-lexer';
```

Comment:
Minor naming affair: we use `hmr-plugin/fine-tune.ts` as the naming convention 👀

Reply:
Done 👀